src/map.rs

```diff
@@ -2492,7 +2492,7 @@ where
                         // we held a guard. Since we found the next pointer in a valid map and
                         // it is not null (as checked above and below), the node it points to was
                         // present (i.e. not removed) from the map while we were marked as active.
-                        // Thus, e cannot be dropped until we release our guard, e is also valid
+                        // Thus, e cannot be dropped until we release our guard. e is also valid
```

Comment:
Hmm, actually, the two instances of this now don't actually say _why_ `e` is valid if it was obtained from a next pointer. I _think_ the argument is the same as what comes before, which is why this used to be a `,`?

Reply:
Ah I understand it now. You were right initially, `"so e is also valid ..." makes more sense.

Reply:
fe8a171148ad6faacfe8d6b791ae93d9ca263905.